DNN Platform/Modules/Journal/ServicesController.cs

```diff
@@ -119,14 +119,31 @@ public HttpResponseMessage Create(CreateDTO postData)
                     postData.ProfileId = UserInfo.UserID;
```

Comment:
You need to remove the ``if`` condition of this block to become basically ``postData.ProfileId = UserInfo.UserID;`` instead of ``if(..) { postData.ProfileId = UserInfo.UserID; }

Reply:
The if condition can't be removed, as have scenario user can post journal on friend's profile page.